src/dao/hooks/types.ts

```diff
@@ -7,6 +7,7 @@ export enum DAOQueries {
   GodTokens = "godTokens",
   FetchLockNFTToken = "fetchLockNFTToken",
   FetchCanUnlockNFTToken = "fetchCanUnlockNFTToken",
+  FetchBlockedTokenBalance = "FetchBlockedTokenBalance",
```

Comment:
value should be in camelcase i.e. "fetchBlockedTokenBalance" to match the other enum values.

Reply:
Done